scripts/release/select_release_assets.py

```
@@ -0,0 +1,157 @@
+def _installer_name_for_artifact(filename: str) -> str:
+    """Return the installer archive filename for a sidecar or archive filename."""
+    for suffix in [".manifest.txt", ".sha256"]:
+        if filename.endswith(suffix):
+            return filename[: -len(suffix)]
+    return filename
+
+
+def _ensure_file(path: Path) -> None:
+    """Raise if a required release asset path is missing or not a file."""
+    if not path.is_file():
+        raise ValueError(f"missing release asset: {path.as_posix()}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)